supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -29,6 +29,26 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 // .Other updates
 // * TODO
 
+[[october-2023]]
+== October 2023
+//
+.Glossary entries
+* xref:ansible-rulebook[Ansible Rulebook] Added glossary entry
+* because: Removed glossary entry because guidance already exists in the _IBM Style_ guidance
+* xref:executable-adj[executable (adjective)] Added glossary entry
+* xref:executable[executable (noun)] Added glossary entry
+* xref:pc[PC]: Updated glossary entry to remove mention of the _IBM Style_ guide
+* xref:playbook[playbook] Added glossary entry
+* xref:rulebook[rulebook] Added glossary entry
+* xref:segmentation-fault[segmentation fault]: Updated glossary entry for clarity
+* xref:spec-file[spec file]: Updated glossary entry for clarity
+//
+//.Style guidance
```

Comment:
Not critical, but I think you could remove these commented-out sections just to keep things clean